ctadiv to divtel

takdg123 opened the pull request:
this contains all my updates.

Commit:
add ctadiv and examples

Commit:
ctadiv to divtel

Commit:
edit env

Commit:
edit env

Commit:
edit env

maxnoe commented on divtel/ArrayConfig/pointing.py:
```diff
@@ -0,0 +1,127 @@
+from astropy.coordinates import SkyCoord, ICRS
+
+
+def alt_az_to_vector(alt, az):
```
Astropy coordinates directly support cartesian coordinates (also works for altaz)

```
In [1]: from astropy.coordinates import SkyCoord

In [2]: c = SkyCoord.from_name("Crab Nebula")

In [3]: c.cartesian
Out[3]: 
<CartesianRepresentation (x, y, z) [dimensionless]
    (0.10280914, 0.92137099, 0.37484102)>
```

Commit:
check

vuillaut commented on divtel/ArrayConfig/pointing.py:
```diff
@@ -0,0 +1,127 @@
+    G: array
+        [Gx, Gy, Gz]
+
+    Returns
+    -------
+    alt: float
+        altitude
+    az: float
+        azumith angle
+    """
+    GT = np.sqrt(((tel_position - G) ** 2).sum())
+    alt = np.arcsin((tel_position[2] - G[2]) / GT)
+    az = np.arctan2(-(tel_position[1] - G[1]), (tel_position[0] - G[0]))
+    return alt, az
+
+
+def pointing_coord(table, frame, icrs=False):
+    """
+    Sky coordinate of each telescope with a given frame.
+
+    Parameters
+    ----------
+    table: astropy.table
+        table containing `alt` and `az` of each telescope
+    frame: class.CTA_Info
+        frame containing information of site and obsrvation time
+    icrs: bool, optional
+        If True, it returns ICRS coordinate
+
+    Returns
+    -------
+    astropy.coordinates.SkyCoord
+    """
+    coord = SkyCoord(alt=table["alt"], az=table["az"], frame=frame.altaz, unit=table["alt"].unit)
+    
+    if icrs:
+        return coord.transform_to(ICRS())
+    else:
+        return coord
+
```
everything here is not new code - please import from `..pointing`

vuillaut replied:
what's the reason to move everything in a subdir?

takdg123 replied:
Actually, I merged them. They are all in the ArrayConfig.

takdg123 replied:
It is because when I started this project, there was another sub module about the multiplicity. So I made a subfolder. However, the multiplicity stuff was not fully developed.

takdg123 replied:
I think we can remove the subfolder if you want.

vuillaut reviewed (changes requested):
Please remove `.ipynb_checkpoints`

Commit:
resolve conflict

Commit:
remove .ipynb_checkpoints

Commit:
remove subdir

Commit:
fix path

Iburelli committed:
Correct telescope phi 

Telescope_phi quantity has been corrected to be consistent with corsika reference frame

Iburelli committed:
create_simtel_config
